connect/config.py

```diff
@@ -28,9 +28,10 @@ class Settings(BaseSettings):
     connect application settings
     """
 
-    # First preference to a defined env var 'LOCAL_CERTS_PATH'
+    # First preference to a defined env var 'APPLICATION_CERT_PATH'
     application_cert_path: str = os.getenv('APPLICATION_CERT_PATH',
                                            os.path.join(Path(__file__).parents[1], 'local-certs'))
+    local_cert_path: str = os.path.join(Path(__file__).parents[1], 'local-certs')
```

Comment:
The NATS NKey file that doesn't match with the installed certs paradigm, so we need one location that actually is always the local-certs directory.  Since APPLICATION_CERT_PATH is './local-certs' when running locally, then the installed cert path when running in a docker container, we can't depend on APPLICATION_CERT_PATH to point to the NKey file.

Reply:
The NATS .nk and .conf files are now in local-certs.  We can still change the name of local-certs as discussed, but there's not really a need to.

Reply:
The names are a bit confusing though. We can think about this a bit in slack, and rename in a follow-up PR. 
I'm thinking that I should have renamed the `local-certs` directory in the project to `application-certs` so that it aligns with the environment variable name. 

Or we just rename the environment variable and host OS directory to local-pki or something and lump the nats conf and "nk" in there as well.

As you can tell I'm thinking and typing at the same time :)